apps/portlet-documents/src/main/resources/locale/portlet/documents_fr.properties

```diff
@@ -44,6 +44,8 @@ Drives.label.DataProtection=Protection des donn\u00E9es
 Drives.label.Guests=Invit\u00E9s
 Drives.label.Management=Gestion
 
+Folder.label.newfolder=Nouveau Dossier
```

Comment:
```suggestion
Folder.label.newfolder=Nouveau dossier
```

What about the english translation ?

Reply:
already added in a previous PR but there was still no crowdin sync but no the sync is done